packages/web-maps/src/components/MapView.tsx

```diff
@@ -20,6 +30,22 @@ class MapView extends React.Component<MapViewProps, State> {
     };
   }
 
+  componentDidMount() {
+    (async () => {
+      if (!this.props.showUserLocation) {
+        return;
+      }
+
+      let { status } = await Location.requestForegroundPermissionsAsync();
+      if (status !== "granted") {
+        return;
+      }
+
+      let location = await Location.getCurrentPositionAsync({});
+      this.setState({ userLocation: location.coords });
```

Comment:
`let` => `const`, also just a suggestion to use the same pattern for unpacking the record fields

```suggestion
      const { status } = await Location.requestForegroundPermissionsAsync();
      if (status !== "granted") {
        return;
      }

      const  { coords } = await Location.getCurrentPositionAsync({});
      this.setState({ userLocation: coords });
```

Reply:
Good call!